### ISSUE TITLE
'Ignore lines on import' bug when import from URL?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#229]
```
What steps will reproduce the problem?
1. import from http://assets.businesslink.gov.uk/transparency/TenderSummary.csv
2. Ignore initial non-blank lines set to 1
3. Import doc

What is the expected output? What do you see instead?
I expect the first line to be ignored, instead the first line is used for 
column headers

What version of the product are you using? On what operating system?
2.0 r 1836. Mac os/x 10.5.8 Google Chrome


```

Original issue reported on code.google.com by `tony.hi...@gmail.com` on 17 Nov 2010 at 12:54

**Comment by GoogleCodeExporter** [comment]
```
Tony, try setting the 'Header lines' to zero.  Does that work as expected?
```

Original comment by `iainsproat` on 17 Nov 2010 at 1:55

**Comment by GoogleCodeExporter** [comment]
```
I am experiencing the same issue. I am importing csv files with no header lines 
and whatever value I input for 'Header lines', Google refine continues to use 
the first row as the header row (I've tried 0,1,2,10,9999999).

I have duplicated the error on two different machines using IE8.
Windows Server 2008 64-bit
Windows 7 64-bit
```

Original comment by `keithwig...@gmail.com` on 7 Jan 2011 at 4:37

**Comment by GoogleCodeExporter** [comment]
```
keithwiggins - It sounds like you are reporting exactly the opposite behavior 
as the original report here.  Can you open a new bug report please?
```

Original comment by `tfmorris` on 7 Jan 2011 at 5:44

**Comment by GoogleCodeExporter** [comment]
```
The problem in the original report appears to be fixed in Refine 2.5.
```

Original comment by `tfmorris` on 20 Dec 2011 at 3:53
- Changed state: **Fixed**
- Added labels: **Milestone-2.5**
